sal_interfaces/DREAM/DREAM_Commands.xml

```diff
@@ -3,61 +3,98 @@
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_setEnabled</EFDB_Topic>
-    <Description>Enable or disable DREAM.</Description>
-    <item>
-      <EFDB_Name>enabled</EFDB_Name>
-      <Description>DREAM enabled or disabled.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>DREAM_command_resume</EFDB_Topic>
+    <Description>Indicate that DREAM is permitted to resume automated operations.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_openHatch</EFDB_Topic>
+    <Description>Open the hatch if DREAM has evaluated that it is safe to do so.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_closeHatch</EFDB_Topic>
+    <Description>Close the hatch.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_stop</EFDB_Topic>
+    <Description>Immediately stop operations and close the hatch.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_operate</EFDB_Topic>
-    <Description>Operate DREAM, meaning taking images, or not.</Description>
+    <EFDB_Topic>DREAM_command_readyForData</EFDB_Topic>
+    <Description>Inform DREAM that Rubin Observatory is ready to receive data or not.</Description>
     <item>
-      <EFDB_Name>operating</EFDB_Name>
-      <Description>DREAM operating, meaning taking images, or not.</Description>
+      <EFDB_Name>ready</EFDB_Name>
+      <Description>Ready to receive data or not.</Description>
```

Comment:
If the Rubin Observatory is not ready yet, why does it want to issue the  `DREAM_command_readyForData` with the `ready=False`?

Reply:
Yes,the command has an interesting name. In general `ready=True` will be sent. If there is some internal problem (network to the rest of the world down for instance) then `ready=False` will be sent. In that case DREAM will stop sending data (and store it internally) but keep observing.